src/java/htsjdk/samtools/cram/build/ContainerParser.java

```diff
@@ -51,14 +54,16 @@ public ContainerParser(final SAMFileHeader samFileHeader) {
     }
 
     public List<CramCompressionRecord> getRecords(final Container container,
-                                                  ArrayList<CramCompressionRecord> records, ValidationStringency validationStringency) throws IllegalArgumentException,
+                                                  ArrayList<CramCompressionRecord> records, final ValidationStringency validationStringency) throws IllegalArgumentException,
             IllegalAccessException {
         final long time1 = System.nanoTime();
-        if (records == null)
+        if (records == null) {
             records = new ArrayList<CramCompressionRecord>(container.nofRecords);
+        }
 
-        for (final Slice slice : container.slices)
+        for (final Slice slice : container.slices) {
             records.addAll(getRecords(slice, container.header, validationStringency));
+        }
 
         final long time2 = System.nanoTime();
```

Comment:
Can we remove this instrumentation (maybe in another PR) ? There is quite a bit of it in the cram code.